prowler/providers/github/services/organization/organization_default_repository_permission_strict/organization_default_repository_permission_strict.metadata.json

```
@@ -0,0 +1,32 @@
+{
+  "Provider": "github",
+  "CheckID": "organization_default_repository_permission_strict",
+  "CheckTitle": "Ensure strict base repository permissions are set for the organization",
+  "CheckType": [],
+  "ServiceName": "organization",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "",
+  "Severity": "high",
+  "ResourceType": "GitHubOrganization",
+  "Description": "Ensure the organization's base repository permission for members is set to 'read' or 'none' to minimize risk.",
+  "Risk": "If base repository permissions allow 'write' or 'admin' by default, organization members may unintentionally gain excessive privileges across repositories, increasing the risk of unauthorized changes or accidental modifications.",
+  "RelatedUrl": "https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/managing-base-permissions-for-an-organization",
```

This URL is returning 404, I think this is the proper one:

```suggestion
  "RelatedUrl": "https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/setting-base-permissions-for-an-organization",
```